recipe/meta.yaml

```
@@ -36,6 +36,8 @@ requirements:
     - point-annotator
 
 test:
+  requires:
+    - python {{ python_min }}
   commands:
      # - pwd
```

```suggestion
    - python {{ python_min }}
    - pip
  commands:
     # - pwd
     - pip check
```
https://github.com/conda-forge/orange3-bioinformatics-feedstock/pull/37/files